internal/slicer/slicer.go

```diff
@@ -82,7 +82,8 @@ func Run(options *RunOptions) error {
 					sourcePath = targetPath
 				}
 				extractPackage[sourcePath] = append(extractPackage[sourcePath], deb.ExtractInfo{
-					Path: targetPath,
+					Path:     targetPath,
+					Optional: pathInfo.Optional,
```

Comment:
Having setup with Optional is slightly unexpected because we don't actually want to expose that as an option to the user, which is mainly what setup is about.  Wouldn't it work if we just added the copyright file here as an optional file for extraction, despite the setup?

Reply:
Good idea. Moved the handling of copyright files from setup to slicer.